.github/workflows/build-cli.yml

```diff
@@ -27,6 +27,21 @@ jobs:
     runs-on: ${{ inputs.os }}
     steps:
       - uses: actions/checkout@v2
+      - name: Cache turbo
+        uses: actions/cache@v4
+        with:
+          path: .turbo
+          key: ${{ runner.os }}-turbo-${{ inputs.artifact-name }}-${{ github.sha }}
+          restore-keys: |
+            ${{ runner.os }}-turbo-${{ inputs.artifact-name }}-
```

Comment:
TBH I don't really understand what's happening here, so apologies for the basic question, but should we have `${{ inputs.artifact-name }}-${{ github.sha }}` or the like here?

Reply:
This restores `.turbo` directory that contains all the `hashed` `inputs` and `outputs`.

The `actions/cache` will lookup `key` first, then if it's not available it will look for entry starting with the patterns provided in `restore-keys`.
So this let's us restore cache from previous commit.